src/bundle/Controller/ContentTypeController.php

```diff
@@ -710,7 +713,9 @@ public function viewAction(ContentTypeGroup $group, ContentType $contentType): R
         $canUpdate = $this->isGranted(new Attribute('class', 'update')) &&
             $this->isGranted(new Attribute('class', 'create'));
 
-        return $this->render('@ibexadesign/content_type/index.html.twig', [
+        $view = $request->get('view_template') ?? '@ibexadesign/content_type/index.html.twig';
```

Comment:
Use specific parameter source. Otherwise this allows control over what templates are being rendered by the malicious agent through URL query parameters, post data or cookies.

```suggestion
        $view = $request->attributes->get('view_template') ?? '@ibexadesign/content_type/index.html.twig';
```